chat-plugins/github.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * GitHub Alert Notifications
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * This notifies a set of rooms of changes to specific repositor(y/ies).
+ *
+ * Much of this comes from Ecuacion's bot, which is based on xfix's bot,
+ * and jd who originally converted this to PS server-side.
+ *
+ * @license MIT license
+ */
+
+'use strict';
+
+let gitConfig = {};
+
+if (!Config.github) {
+	return;
+} else if (Config.github === {}) {
+	gitConfig.port = 3420;
+	gitConfig.secret = "";
+} else {
+	gitConfig = {
+		port: Config.github.port,
+		secret: Config.github.secret,
+	};
+}
+
+const git = exports.github = require('githubhook')(gitConfig);
```

Comment:
Actually, why not configure Config.github to be something like this:

```
exports.github = {
    rooms: [],
    options: {
        port: 3420,
        secret: '',
    },
};
```

Then you could just ensure that `Config.github` has some fallback values if its not set, then pass `Config.github.options` instead of `gitConfig` here. Making a second config object for this is unnecessary